src/sentry/static/sentry/app/views/organizationIntegrations/sentryAppInstallations.jsx

```diff
@@ -73,9 +73,7 @@ class SentryAppInstallations extends React.Component {
   };
 
   openModal = app => {
-    const {organization} = this.props;
-    const onInstall = () => this.install(app);
-    openSentryAppPermissionModal({app, onInstall, orgId: organization.slug});
```

Comment:
@MeredithAnya Doesn't seem like we need `openSentryAppPermissionModal` anymore, right? If so, I think we should clean up the code.

Reply:
Yeah that's why I put WIP, gotta update test and remove code

Reply:
@MeredithAnya If this is WIP, why not just create a PR draft instead? The bit about the WIP was buried.

Reply:
wanted to let the tests run see what was broken. I can add it in the title next time and not just a label.

Reply:
@MeredithAnya FYI, if you create a PR draft the tests will still run

Reply:
oof

Reply:
Maybe I just love my PR drafts too much